cogs/modmail.py

```diff
@@ -2268,31 +2373,125 @@ async def isenable(self, ctx):
     @checks.thread_only()
     async def snooze(self, ctx, *, duration: UserFriendlyTime = None):
         """
-        Snooze this thread: deletes the channel, keeps the ticket open in DM, and restores it when the user replies or a moderator unsnoozes it.
-        Optionally specify a duration, e.g. 'snooze 2d' for 2 days.
-        Uses config: max_snooze_time, snooze_title, snooze_text
+        Snooze this thread. Behavior depends on config:
+        - delete (default): deletes the channel and restores it later
+        - move: moves the channel to the configured snoozed category
+            Optionally specify a duration, e.g. 'snooze 2d' for 2 days.
+            Uses config: snooze_default_duration, snooze_title, snooze_text
         """
         thread = ctx.thread
         if thread.snoozed:
             await ctx.send("This thread is already snoozed.")
             logging.info(f"[SNOOZE] Thread for {getattr(thread.recipient, 'id', None)} already snoozed.")
             return
-        max_snooze = self.bot.config.get("max_snooze_time")
-        if max_snooze is None:
-            max_snooze = 604800
-        max_snooze = int(max_snooze)
+        from core.time import ShortTime
+
+        default_snooze = self.bot.config.get("snooze_default_duration")
+        if default_snooze is None:
+            default_snooze = 604800
+        else:
+            try:
+                default_snooze = int(default_snooze)
+            except (ValueError, TypeError):
+                default_snooze = 604800
         if duration:
             snooze_for = int((duration.dt - duration.now).total_seconds())
-            if snooze_for > max_snooze:
-                snooze_for = max_snooze
+            if snooze_for > default_snooze:
+                snooze_for = default_snooze
```

Comment:
why cap the snooze_for to the maximum of the default_snooze? Also if this has good reason, a better line would be good: `snooze_for = min(snooze_for, default_snooze)`